app/services/import/legacy/fetch-company-contacts.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+/**
+ * Fetches the company data from the import.NALD_PARTIES table for the licence ref
+ * @module FetchCompanyContactsService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Fetches the licence data from the import.NALD_PARTIES table for the licence ref
+ *
+ * From this point parties will be referred to as companies and a party will be known as a company
+ *
+ * @param {string} regionCode - The NALD region code
+ * @param {string} licenceId - The NALD licence ID
+ *
+ * @returns {Promise<ImportLegacyCompanyContactType[]>}
+ */
+async function go (regionCode, licenceId) {
+  const query = _query()
+
+  const { rows } = await db.raw(query, [regionCode, licenceId, regionCode, licenceId])
+
+  return rows
+}
+
+function _query () {
+  return `
+  SELECT DISTINCT ON (np."ID")
+    (concat_ws(':', np."FGAC_REGION_CODE", np."ID")) AS external_id,
+    (
+      CASE np."SALUTATION"
+        WHEN 'null' THEN NULL
+        ELSE np."SALUTATION"
+        END
+      )  AS salutation,
+    (
+      CASE np."INITIALS"
+        WHEN 'null' THEN NULL
+        ELSE np."INITIALS"
+        END
+      )  AS initials,
+    (
+      CASE np."FORENAME"
+        WHEN 'null' THEN NULL
+        ELSE np."FORENAME"
+        END
+      )  AS firstName,
+    (
+      CASE np."NAME"
+        WHEN 'null' THEN NULL
+        ELSE np."NAME"
+        END
+      )  AS lastName
+  FROM import."NALD_PARTIES" np
```

Comment:
Need to add logic to ignore if the type is ORG. 

Old import logic - 

```
/**
 * Maps NALD party to CRM contact
 * @param {Object} NALD party
 * @return {Object}
 */
const mapContact = party => {
  if (party.APAR_TYPE === 'ORG') {
    return null
  }
  return {
    salutation: str.mapNull(party.SALUTATION),
    initials: str.mapNull(party.INITIALS),
    firstName: str.mapNull(party.FORENAME),
    lastName: str.mapNull(party.NAME),
    externalId: `${party.FGAC_REGION_CODE}:${party.ID}`,
    _nald: party
  }
}
```

Reply:
One thing to note is that company only has a contact id it has a licence version and the type is not organisation.